packages/utilities/psammead-styles/src/typography.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shape, func } from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import {
+  getCanon,
+  getTrafalgar,
+  getParagon,
+  getDoublePica,
+  getGreatPrimer,
+  getBodyCopy,
+  getPica,
+  getLongPrimer,
+  getBrevier,
+  getMinion,
+} from '@bbc/gel-foundations/typography';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import styled from 'styled-components';
+
+const TypographyText = styled.p`
+  ${props =>
+    props.script && props.typographyFunc
+      ? props.typographyFunc(props.script)
+      : ''}
+`;
+
+TypographyText.propTypes = {
+  script: shape(scriptPropType).isRequired,
+  typographyFunc: func.isRequired,
+};
+
+const typographyStory = typographyFunc =>
+  inputProvider([{ name: 'sample text' }], ([text], script) => (
+    <TypographyText script={script} typographyFunc={typographyFunc}>
+      {text}
+    </TypographyText>
+  ));
+
+storiesOf('Typography', module)
+  .addDecorator(withKnobs)
+  .add('Canon', typographyStory(getCanon))
```

Comment:
If you do 
```import * as typographies from '@bbc/gel-foundations/typography';```

You can then do 
```
const stories = storiesOf('Typography', module)
  .addDecorator(withKnobs)
```

then
```
Object.keys(typographies).forEach(typographyName => {
	stories.add(typographyName, typographyStory(typographies[typographyName]))
});
```

IMO its a bit nicer, and means when we add more, we dont have to manually add them here :)